spark/src/test/scala/org/apache/spark/sql/delta/MergeIntoDVsSuite.scala

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (2024) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.sql.delta
+
+import org.apache.spark.sql.delta.cdc.MergeCDCTests
+import org.apache.spark.sql.delta.sources.DeltaSQLConf
+
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.sql.types.StructType
+
+trait MergeIntoDVsTests extends MergeIntoSQLSuite with DeletionVectorsTestUtils {
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    spark.conf.set(DeltaConfigs.ENABLE_DELETION_VECTORS_CREATION.defaultTablePropertyKey, "true")
+    spark.conf.set(DeltaSQLConf.MERGE_USE_PERSISTENT_DELETION_VECTORS.key, "true")
```

Comment:
There is a util class https://github.com/delta-io/delta/blob/master/spark/src/test/scala/org/apache/spark/sql/delta/DeletionVectorsTestUtils.scala controlling which DV to use. Could you move these two lines to that file?